internal/validation/testdata/gen.go

```diff
@@ -0,0 +1,9 @@
+// Package testdata copies validation test cases from the Javascript reference implementation.
+//
+// Usage:
```

Comment:
I had a lot of trouble figuring out how to use this package. I wasn't able to get all the way there because `babel-node` doesn't respect the `graphqljs/.babelrc` settings.

I'd like some help documenting the actual steps needed to update the validation test data.